PowerPointLabs/PowerPointLabs/ActionFramework/EffectsLab/MakeTransparent/MakeTransparentActionHandler.cs

```diff
@@ -23,7 +23,7 @@ protected override void ExecuteAction(string ribbonId)
 
             if (selection.Type != PowerPoint.PpSelectionType.ppSelectionShapes)
             {
-                MessageBox.Show("Please select at least 1 shape");
+                MessageBox.Show("Please select at least 1 shape.");
```

Comment:
Since you are already changing these, why not add it to TextCollection too? For all the rest too.